pkg/skaffold/initializer/config_test.go

```diff
@@ -239,3 +186,24 @@ func Test_canonicalizeName(t *testing.T) {
 		})
 	}
 }
+
+func artifacts(pairs []build.BuilderImagePair) []*latest.Artifact {
```

Comment:
This function is very copy of  https://github.com/GoogleContainerTools/skaffold/blob/45a55e159b46b5a3ba0c0bdd94f6376f1997eb7d/pkg/skaffold/initializer/build/util.go#L84 and can be removed too.